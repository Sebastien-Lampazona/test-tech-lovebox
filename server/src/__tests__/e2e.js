const { gql } = require('apollo-server');
const { createTestClient } = require('apollo-server-testing');
const { ApolloServer, schema } = require('../server');

const server = new ApolloServer({ ...schema });
const GET_USERS = gql`query {
  users {
    email
    role {
      name
    }
  }
}`
describe('Test Server, no context', () => {
  it('Users whit out credentials', async () => {
    const { query } = createTestClient(server);
    const res = await query({ query: GET_USERS });
    expect(res).toMatchSnapshot();
  });
});
