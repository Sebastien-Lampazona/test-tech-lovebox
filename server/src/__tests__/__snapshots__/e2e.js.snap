// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Server, no context Users whit out credentials 1`] = `
Object {
  "data": Object {
    "users": null,
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
